packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/ClusterManagersController.java

```diff
@@ -89,11 +102,22 @@ void addClusterManagers(@NonNull List<Messages.PlatformClusterManager> clusterMa
   void addClusterManager(String clusterManagerId) {
     ClusterManager<MarkerBuilder> clusterManager =
         new ClusterManager<MarkerBuilder>(context, googleMap, markerManager);
-    ClusterRenderer<MarkerBuilder> clusterRenderer =
-        new ClusterRenderer<MarkerBuilder>(context, googleMap, clusterManager, this);
+    initializeRenderer(clusterManager);
+    clusterManagerIdToManager.put(clusterManagerId, clusterManager);
+  }
+
+  /**
+   * Initializes cluster renderer if it hasn't been initialized before. ClusterManager uses default
```

Comment:
This sounds like it would be a no-op if it had already been called, but that doesn't appear to be the case in the implementation.